.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -290,6 +291,12 @@ jobs:
       if: inputs.tests_selector == 'cts'
       env:
         ONEAPI_DEVICE_SELECTOR: ${{ inputs.target_devices }}
+      # FIXME: For some reason the "sub_group api" test-case is failing with
+      # SIGSEGV while running test_all, so running each binary separately.
+      # BTW test_all requires a lot of resources to build it, so probably it'll
```

Comment:
Should we try per-source kernel splitting maybe?

Reply:
I can't imagine other reasons for the slowdown...

Reply:
`per-source` splitting is a default mode of splitting